telemetry/server.go

```diff
@@ -48,16 +48,24 @@ func (s *Server) createReceivedMessages(w http.ResponseWriter, r *http.Request)
 	}
 	defer r.Body.Close()
 
-	if err := receivedMessage.put(s.DB); err != nil {
-		log.Println(err)
+	var ids []int
+	for _, receivedMessage := range receivedMessages {
+		if err := receivedMessage.put(s.DB); err != nil {
+			log.Println(err)
```

Comment:
Adding the receivedMessage Id in this log could be helpful for debugging

Reply:
ID is not present cause not saved ;) but I added the message itself to the log